buffer/folder.tf

```diff
@@ -0,0 +1,17 @@
+locals {
+  create_folders = var.enable && var.parent_folder_id != ""
+}
+
+resource "google_folder" "buffer_project_folder" {
+  count        = local.create_folders ? 1 : 0
+  display_name = "${local.service}-${local.owner}"
+  parent       = "folders/${var.parent_folder_id}"
+  provider     = google.target
+}
+
+resource "google_folder" "buffer_per_pool_folders" {
```

Comment:
maybe just "pool_folders"? "buffer_per" seems a little redundant in this context. Also maybe worth a short comment.

Reply:
Done.